src/controllers/director.correspondence.address.manual.controller.ts

```diff
@@ -158,3 +159,22 @@ export const buildValidationErrors = (validationStatusResponse: ValidationStatus
 
   return validationErrors;
 }
+
+export const renderPage = (req: Request, res: Response, serviceAddress: Address, officerFiling: OfficerFiling, validationErrors: ValidationError[]) => {
+  const formattedErrors = formatValidationErrors(validationErrors);
+  return res.render(Templates.DIRECTOR_CORRESPONDENCE_ADDRESS_MANUAL, {
+    templateName: Templates.DIRECTOR_CORRESPONDENCE_ADDRESS_MANUAL,
+    backLinkUrl: urlUtils.getUrlToPath(DIRECTOR_CORRESPONDENCE_ADDRESS_SEARCH_PATH, req),
+    directorName: formatTitleCase(retrieveDirectorNameFromFiling(officerFiling)),
+    typeahead_array: COUNTRY_LIST,
+    correspondence_address_premises: serviceAddress.premises,
+    correspondence_address_line_1: serviceAddress.addressLine1,
+    correspondence_address_line_2: serviceAddress.addressLine2,
+    correspondence_address_city: serviceAddress.locality,
+    correspondence_address_county: serviceAddress.region,
+    typeahead_value: serviceAddress.country,
```

Comment:
I am confused here and don't know if it is a requirement but why is the country called typeahead_value?

Reply:
Hi @okoye-companieshouse , I have _not_ changed the functionality of this block of code, if you notice I've moved block implementation in `renderPage()`method (copied code from 82-97).